readthedocs/core/static/core/js/integrations.js

```diff
@@ -0,0 +1,44 @@
+// Unique entry point that our servers will inject
+
+// Load Read the Docs data first
+// This data is generated at build time by the doctool
+fetch("/en/manual-integration-docsify/readthedocs-data.html", {method: 'GET'})
```

Comment:
Noted below, but we could probably just use JSON here.

Also, [to expand on below](https://github.com/readthedocs/readthedocs.org/pull/10127/files#r1132952089), ultimately, these features won't be loaded as _global scripts_ [^1], but as modules in this package.

To illustrate, without getting into asynchronous imports:

```js
import { DocDiff } from 'readthedocs-docdiff';
import { DocEmbed } from 'readthedocs-embed';

fetch("/blah/readthedocs-data.json")
   ...
  .then((readthedocs_data) => {
    const doc_diff = new DocDiff(readthedocs_data);
    doc_diff.enable();
    const doc_embed = new DocEmbed(readthedocs_data);
    doc_embed.enable();
    ...
  }
}
```
   
[^1]: Global scripts are loaded on to window/global, and we have less control over what these are. Instead, our package should isolate itself _and all dependencies_ in a bundle.